mods/ts/rules/defaults.yaml

```diff
@@ -227,13 +227,21 @@
 		TargetTypes: Ground, Infantry
 	RenderSprites:
 	WithInfantryBody:
-	WithDeathAnimation:
+	WithDeathAnimation@normal:
 		DeathTypes:
 			BulletDeath: 1
 			SmallExplosionDeath: 2
-			ExplosionDeath: 3
-			InfantryExplodeDeath: 4 #not used by any warhead; TODO: check what should use this type and add it!
+			HeadshotDeath: 3 #not used by any warhead; TODO: check what should use this type and add it!
```

Comment:
Something is wrong here.
You are removing the currently unused death type, but keeping the "This is not used!" comment on the used type.